DeveloperDashboardClient/DataServices/GitClient.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Octokit;
+using System.Reflection;
+
+namespace DeveloperDashboardClient.DataServices
```

Comment:
Create a separate folder called "Clients" in the solution and move this class in it